tests/annotations/tests.py

```diff
@@ -188,6 +271,15 @@ def test_aggregate_over_annotation(self):
         other_agg = Author.objects.aggregate(age_sum=Sum('age'))
         self.assertEqual(agg['otherage_sum'], other_agg['age_sum'])
 
+    def test_aggregate_over_alias(self):
+        with self.assertRaisesMessage(
+            FieldError,
+            "Cannot aggregate over the 'other_age' alias. Use annotate() to promote it"
+        ):
```

Comment:
```suggestion
        msg = (
            "Cannot aggregate over the 'other_age' alias. Use annotate() to "
            "promote it."
        )
        with self.assertRaisesMessage(FieldError, msg):
```
Use the same pattern in other `assertRaisesMessage`.